examples/gno.land/p/demo/grc/grc20/z_0_filetest.gno

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	banker "gno.land/p/demo/grc/grc20"
+	"gno.land/p/demo/testutils"
+)
+
+const uint64_max = 1<<64 - 1
+
+func main() {
+	b := banker.NewBanker("Dummy", "DUMMY", 6)
+	var owner = testutils.TestAddress("owner")
+	// Mint uint64_max - 1000 tokens for owner
+	b.Mint(owner, uint64_max-1000)
+	// Try to mint 1001 tokens for owner, should fail because it exceeds the limit
+	println(b.Mint(owner, 1001).Error())
+
+}
+
+// Output:
+// you can't mint more than 1000 tokens
```

Comment:
To be honest, filetests should be discouraged. We should strive to write proper tests, that are more Go idiomatic. 
They are more readable & more understandable by the broader community, and provide actual type checking on the errors. On the other hand, filetests are supereasy to write, but in the end just provide a println output.